src/fdb5/api/SelectFDB.cc

```diff
@@ -72,7 +85,7 @@ SelectFDB::SelectFDB(const Config& config, const std::string& name) : FDBBase(co
     }
 
     for (const auto& c : config.getSubConfigs("fdbs")) {
-        subFdbs_.emplace_back(std::make_pair(parseFDBSelect(c), FDB(c)));
+        subFdbs_.emplace_back(FDBLane{parseFDBSelect(c), c, std::optional<FDB>{}});
```

Comment:
I personally prefer to use `std::nullopt` here instead of `std::optional<FDB>{}}` as I think this is easier to parse for the reader.